commands/provider/marketplace_types.go

```diff
@@ -54,11 +54,11 @@ type MarketplaceDeployedSource struct {
 
 // MarketplaceManifestServiceData is the data present to the manifest and sent to create a new service's version
 type MarketplaceManifestServiceData struct {
-	Definition  importer.ServiceDefinition `json:"definition"`
-	Readme      string                     `json:"readme,omitempty"`
-	Hash        string                     `json:"hash"`
-	HashVersion string                     `json:"hashVersion"`
-	Deployment  MarketplaceDeployedSource  `json:"deployment"`
+	Definition  *coreapi.Service          `json:"definition"`
```

Comment:
use `definition.Service`?

Reply:
good catch, that's because we merged this but still kept the service also in the definition and this PR was done before that... fixed